src/app/About/AboutCryostatModal.tsx

```diff
@@ -51,7 +53,7 @@ export const AboutCryostatModal = ({ isOpen, onClose }) => {
         brandImageAlt="Cryostat Logo"
         isOpen={isOpen}
         onClose={onClose}
-        trademark={CRYOSTAT_TRADEMARK}
+        trademark={t('CRYOSTAT_TRADEMARK') as string}
```

Comment:
Is the `as string` really needed here? I'd hope that the type signature is `t(msg: string): string` :-/

Reply:
Seems like the problem is that the function potentially returns `null` which is not assignable to `string | undefined`.
Doing digging, I can solve it through this tutorial. https://www.i18next.com/overview/typescript#argument-of-type-defaulttfuncreturn-is-not-assignable-to-parameter-of-type-xyz

Reply:
Ahh, that makes more sense.